performance_regression_test.py

```diff
@@ -654,3 +657,36 @@ def test_timeseries_bench(self):
         self.display_results(results, test_name='test_timeseries_read_bench')
         self.check_regression()
         self.kill_stress_thread()
+
+    def test_mixed_with_stairs(self):
```

Comment:
what are the "stairs"?

Reply:
it means that ops will increase on each step like steps on the stairs. But you right it should be renamed